tests/algorithms/test_sqil.py

```diff
@@ -1,4 +1,6 @@
 """Tests `imitation.algorithms.sqil`."""
+from unittest.mock import MagicMock
```

Comment:
Nit: violates our style guide https://google.github.io/styleguide/pyguide.html#22-imports
> Use import statements for packages and modules only, not for individual classes or functions.
```suggestion
from unittest import mock
```

Reply:
(good use of mocking though!)

Reply:
thanks for pointing it out. I hadn't realized that convention.